_layouts/post.html

```diff
@@ -81,14 +81,14 @@ <h5 class="index-headline featured"><span>Written by</span></h5>
               <p class="published">Published <time datetime="{{ page.date | date: '%F %R' }}">{{ page.date | date_to_string }}</time></p>
             </section>
           </div>
-          {{/post}}
+          {{post}}
           <div class="isRight">
             <h5 class="index-headline featured"><span>Supported by</span></h5>
             <footer class="site-footer">
               <section class="poweredby">Proudly published with <a href="http://jekyllrb.com"> Jekyll</a></section>
               <a class="subscribe" href="{{ "/feed.xml" | prepend: site.baseurl }}"> <span class="tooltip"> <i class="fa fa-rss"></i> You should subscribe to my feed.</span></a>
               <div class="inner">
-                <section class="copyright">All content copyright <a href="{{@blog.url}}/">{{site.author}}</a> &copy; {{ site.time | date: '%Y' }}<br>All rights reserved.</section>
+                <section class="copyright">All content copyright <a href="{{blog.url}}/">{{site.author}}</a> &copy; {{ site.time | date: '%Y' }}<br>All rights reserved.</section>
```

Comment:
I believe that came with the theme that I copied & modified. I didn't get deep enough to change variables, I believe. Thank you so much for your help. I'm going to merge this pull.